db/seeds.rb

```diff
@@ -4,6 +4,6 @@
 
 gds_organisation_content_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
 
-user.update!(permissions: [User::PRE_RELEASE_FEATURES_PERMISSION],
+user.update!(permissions: [User::PRE_RELEASE_FEATURES_PERMISSION, User::DEBUG_PERMISSION],
```

Comment:
I'm a bit concerned about the test user having debug permission by default. Could we do that if you're in a dev environment?

Could this be a separate commit too?

Reply:
Fair enough - done :-).